software/externals/cmake/externals.cmake

```diff
@@ -142,7 +142,7 @@ set(flycapture_revision be27acc7effd83b0cb5648742b915eb4a6181f49)
 set(flycapture_depends)
 set(flycapture_external_args
   ${download_only_args}
-  SOURCE_DIR ${source_prefix}/../drivers/flycapture
+  SOURCE_DIR ${source_prefix}/../atlas-collection/flycapture
```

Comment:
Even though flycapture is used on atlas, couldn't it also be used on other robots?  Why not apply the same ExternalProject logic to replace drivers/tobuild.txt?  We can admit that tobuild.txt is not sufficient and start fixing it.